website/src/assets/images/components/checkbox/sl-checkbox-anatomy.svg

```diff
@@ -0,0 +1,26 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 656 224" fill="none">
+  <g clip-path="url(#clip0_2320_6652)">
```

Comment:
@arecuenco-dsgn can you please also prepare descriptions of SVGs for the screen readers (sth we discussed during our last retrospective).

Reply:
descriptions added